modules/cas_disk/disk.h

```diff
@@ -31,6 +31,10 @@ struct casdsk_disk {
 
 	struct mutex lock;
 
+	struct mutex openers_lock;
```

Comment:
You need to initialize this mutex before using it

Reply:
it is initalized in disk.c:129, right after dsk->lock initialization.